umami/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: umami_app_1
+      APP_PORT: 3000
+      PROXY_AUTH_ADD: "false"
+  
+  app:

```

Comment:
Looking at the[ Dockerfile for Umami](https://github.com/umami-software/umami/blob/7a75639dd3d7aeff46104b71ebfb3853fc0eee09/Dockerfile#L54), we can easily override the default behaviour of this container to run as the `umbrel` user 1000 by adding this line:

`user: "1000:1000"`